reference-architecture/osp-cli/haproxy.cfg.template

```diff
@@ -0,0 +1,63 @@
+global
+    log         127.0.0.1 local2
+
+    chroot      /var/lib/haproxy
+    pidfile     /var/run/haproxy.pid
+    maxconn     4000
+    user        haproxy
+    group       haproxy
+    daemon
+    # turn on stats unix socket
+    stats socket /var/lib/haproxy/stats
+
+defaults
+    log                     global
+    option                  httplog
+    option                  dontlognull
+    option http-server-close
+    option                  redispatch
+    retries                 3
+    timeout http-request    10s
+    timeout queue           1m
+    timeout connect         10s
+    timeout client          1m
+    timeout server          1m
+    timeout http-keep-alive 10s
+    timeout check           10s
+    maxconn                 3000
+listen  stats :9000
+    	stats enable
+    	stats realm Haproxy\ Statistics
+        stats uri /haproxy_stats
+        stats auth admin:password 
+		stats refresh 30
+		mode http
+
+frontend  main *:80
+    default_backend             router80
+
+backend router80
+    balance source
+    mode tcp
+    # INFRA_80
+    #server infra-node-0.%OCP3_DOMAIN% %INFRA_0_IP%:80 check
+
+
+frontend  main *:443
+    default_backend             router443
+
+backend router443
+    balance source
+    mode tcp
+    # INFRA_443
+    #server infra-node-0.%OCP3_DOMAIN% %INFRA_0_IP%:443 check
+
+
+frontend  main *:8443
```

Comment:
That would conflict with the app inbounds on port 443, would it not? With a single TCP haproxy, the app and master connections inbound must be on different ports to be forwarded to different hosts.